docs/source/guide/labeling.md

```diff
@@ -115,7 +115,7 @@ To label tasks multiple times while the minimum annotations required is set to o
 5. Click the next task in the data manager to open the quick labeling view for that task and repeat steps 2-4.
 
 
-## Perform ML-assisted labeling with interactive preannotations
```

Comment:
This header should contain "interactive preannotations" anyway, why did you remove it?